packages/zwave-js/src/lib/driver/Driver.ts

```diff
@@ -6351,6 +6354,7 @@ ${handlers.length} left`,
 					node.updateRTT(msg);
 					// Update last seen state
 					node.lastSeen = new Date();
+					node.updateRate();
```

Comment:
We shouldn't update the rate here. This codepath is executed when an outgoing command is acknowledged. We're only interested in the rate of incoming commands, which should be low after the interview is completed.